backend/internal/storage/storage.go

```diff
@@ -27,6 +27,8 @@ type Assignments interface {
 	CreateSession(ctx context.Context, sessionData models.Session) error
 	GetSession(ctx context.Context, gitHubUserID int64) (models.Session, error)
 	DeleteSession(ctx context.Context, gitHubUserID int64) error
+  GetAssignmentIDs(ctx context.Context) ([]models.Assignment_CR_ID, error)
```

Comment:
I'm not a fan of this name. Instead of saying CR, I think github would be better here. CR is not immediately obvious

Reply:
going to say Assignment_Classroom_ID to specify github classroom and not github